shell/platform/embedder/embedder.cc

```diff
@@ -14,11 +14,13 @@
 #include "third_party/dart/runtime/bin/elf_loader.h"
 #include "third_party/dart/runtime/include/dart_native_api.h"
 
+#if !defined(EMBEDDER_AS_INTERNAL_LIBRARY)
```

Comment:
Nit: How about just `#ifndef FLUTTER_NO_EXPORT`. Short and to the point.

Reply:
Sure, done.